index.js

```diff
@@ -271,7 +271,7 @@ const inlineCss = async opt => {
     urls: [pageUrl],
     skippable: request =>
       options.skipThirdPartyRequests && !request.url().startsWith(basePath),
-    browser: browser,
+    // browser: browser,
```

Comment:
Any reason not to do this? am I missing something?

Reply:
it will launch two browsers simultaneously (instead of one).

I wonder why it works. What minimalcss does so its own instance actually disables serviceWorkers

Reply:
After going down the rabbit hole and if I have to guess, it seems that there was an issue in puppeteer itself, page.goto returning null in some cases
https://github.com/GoogleChrome/puppeteer/issues/2479
https://github.com/GoogleChrome/puppeteer/issues/1391
And I think that disabling service workers solved it in minimalcss

if thats the case maybe trying to use the same browser with service workers both for react-snap and minimalcss

but I don't really now, this is speculative.
I also don't understand why minimalcss is calling puppeteer.launch for each page instead of puppeteer.connect or something like that - that might make it reasonable to have one browser for react-snap and one for minimalcss

Reply:
> I also don't understand why minimalcss is calling puppeteer.launch for each page instead 
because of the API design: it is a function not an instance of object, react-snap calls this function for each page. So if you want to preserve instance of the browser you can pass it as param to the function

The null problem, if I remember it correctly - the problem is that the serviceWorker takes over the network layer and all requests go through it and css files are not served with proper mime type, which is used for detection in this line https://github.com/peterbe/minimalcss/blob/master/src/run.js#L263